tests/nodes/tests_Literal.cpp

```diff
@@ -501,48 +501,48 @@ TEST_CASE("Literal - misc functions") {
         CHECK(99_xsd_int .round() == 99_xsd_integer);
         CHECK(1.2_xsd_double .round() == 1.0_xsd_double);
         CHECK(1.5_xsd_double .round() == 2.0_xsd_double);
-        CHECK("hello"_xsd_string.round().null());
+        CHECK("hello"_xsd_string .round().null());
```

Comment:
What is the plan here? Only viable method working with clang-format seems to be putting parenthesis around. Googled for it but didn't find anything related. 

Could you please fill a bug report for clang-format?

Reply:
Filed a bug: https://github.com/llvm/llvm-project/issues/60576

I also found out that this is only invalid formatting for numeric user-defined-literals but not string user-defined-literals. I guess I'll wrap everything in parentheses so that it will not break by accident in the future

Reply:
And inserted parentheses